bindings/azure/servicebusqueues/servicebusqueues.go

```diff
@@ -154,15 +154,24 @@ func (a *AzureServiceBusQueues) Invoke(ctx context.Context, req *bindings.Invoke
 		a.senderLock.Unlock()
 	}
 
+	a.logger.Infof("Invoke Binding incoming metadata: %+v", req.Metadata)
```

Comment:
Can you make this a debug log please? Especially because the metadata may contain PII.

Reply:
Yes, I actually meant to not include that but it missed my scan so I'm just going to remove it entirely.